src/App.js

```diff
@@ -14,7 +15,8 @@ function App() {
      <div className ="main-container">
       <Header/>
       {/* <HomePage/> */}
-      <WishListPage/>
+      {/* <WishListPage/> */}
```

Comment:
You might use `react-router` here so that you don't have to comment codes and can navigate too. Hope this is intentional here

Reply:
yes this was intentional...I'll use react router here once I complete converting the html pages to jsx